index.html

```diff
@@ -1,9 +1,90 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>NPM Clone</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+      <header>
+        <div>
+            <div>
+              <img class="blackHeart" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\black-heart.png" alt="black heart" />
+              <span>Nifty Penguin Magic</span>
+            </div>
+            <nav id="nav-bar">
+              <ul>
+                <li><a href="#">npm Enterprise</a></li>
+                <li><a href="#">Products</a></li>
+                <li><a href="#">Solutions</a></li>
+                <li><a href="#">Resources</a></li>
+                <li><a href="#">Docs</a></li>
+                <li><a href="#">Support</a></li>
+              </ul>
+            </nav>
+        </div>
+
+        <div>  
+                <img class="logo" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\npm-logo.png" alt="logo"/>  
+              <form class="search-button" action="action_page.php">
+                <img class="search-bar" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\magnifying-glass.png" alt="search bar pic"/>
+                <input type="text" placeholder="Search packages" name="search">
+                <button type="submit" class="search-box">
+                    <i class="fa fa-search">Search</i>
+                 </button>
+              </form>
+                 <div>
+                    <a class="link-btn" id="join-button" href="#">Join</a>
+                    <a class="link-btn" href="#">Log In</a>
+                 </div>
+              
+        </div> 
```

Comment:
I would clean this up a bit in terms of indentations.
Also, I would add a class to this div and style it to be a flex box. Right now this element is wider then all the other and you have to scroll to the side to see it all.